frame/treasury/src/benchmarking.rs

```diff
@@ -16,6 +16,8 @@
 
 //! Treasury pallet benchmarking.
 
+#![cfg(feature = "runtime-benchmarks")]
```

Comment:
Why these changes?

Reply:
imo it is cleaner. do you have an opinion with it?

Reply:
I find it the other way cleaner xD But it is okay.